src/utils.coffee

```diff
@@ -4,13 +4,12 @@ utils =
   base64:
     encode: (data, to_encoding) ->
       return data if to_encoding is 'base64'
-      data = new Buffer data, to_encoding or 'utf8' unless Buffer.isBuffer data
+      data = new Buffer data.toString(), to_encoding or 'utf8' unless Buffer.isBuffer data
```

Comment:
this doesnt seem right, what is data i your case if it is neither a buffer nor a string